template/build/webpack.base.conf.js

```diff
@@ -31,76 +31,73 @@ module.exports = {
       'components': path.resolve(__dirname, '../src/components')
     }
   },
-  resolveLoader: {
-    fallback: [path.join(__dirname, '../node_modules')]
```

Comment:
should not be removed - it's just `resolveLoader.modules` now:

https://webpack.js.org/configuration/resolve/#resolveloader